### PR TITLE
Align installation requirements with terms of service

### PR DESCRIPTION
**Pull request by catherineluse**
Our installation docs had some inconsistencies with the terms of service. This PR updates the installation docs to match the terms of service and addresses this issue https://github.com/rancher/docs/issues/2050

### CODE REVIEW

**Comment by catherineluse** [comment]
I have made two separate section for OS/Docker requirements: one section in the Rancher installation docs, in which Windows server has been removed, and one section in the section for provisioning user clusters, in which Windows server is included.

@superseb Do the terms of service need to be updated to show that version 7.7 is required for RHEL and Oracle Linux?

**Comment by catherineluse** [comment]
I have separated the Docker requirements for Ubuntu into two lines, the way it currently is in the Rancher installation requirements, to make it clear that Docker 17.03.x can be used for Ubuntu 16.04, but not 18.04.

**Comment by catherineluse** [comment]
@sangeethah To make it easier to validate, here is a list of all the existing discrepancies between the Rancher installation requirements and the terms of service.

**Discrepancies in OS Requirements**

OS | OS Version in Installation Doc | OS Version in TOS
----|-----------|-----
RHEL | 7.7 | 7.5, 7.6, 7.7
CENTOS | 7.7 | 7.5, 7.6, 7.7

**Discrepancies in Docker Requirements**

OS | Docker Version in Installation Doc | Docker Version in TOS
----|-------|----
Ubuntu 18.04 | Docker 18.06.x, 18.09.x, 19.03.x | Docker 17.03.2, 18.06.2, 18.09.x, 19.03.x
Oracle Linux 7.6 | Docker 17.03.x, 18.06.x, 18.09.x, 19.03.x | Docker 19.03.x
RancherOS 1.5.4 | Docker 17.03.x, 18.06.x, 18.09.x, 19.03.x | Docker 17.03.2, 18.06.2, 18.09.x (up to 18.09.8), 19.03.x
RHEL 7.7 in install docs, or 7.5, 7.6, 7.7 in TOS | RHEL Docker 1.13 | "RHEL Docker 1.13.x
Docker 17.03.2, 18.06.2, 18.09.x, 19.03.x"
CentOS 7.7 in install docs, or 7.5, 7.6, 7.7 in TOS | RHEL Docker 1.13 | Docker 17.03.2, 18.06.2, 18.09.x, 19.03.x

**Comment by catherineluse** [comment]
Update: After talking to Denise, it sounds like the table of OS and Docker requirements should be removed from the installation docs, and we should just say:

> Rancher should work with any modern versions of Linux and Docker. For details on which OS and Docker version Rancher was tested on, refer to Rancher's terms of service.

File: content/rancher/v2.x/en/cluster-provisioning/node-requirements/_index.md [hunk 1]
```diff
@@ -7,17 +7,37 @@ This page describes the requirements for the nodes where your apps and services
...
 - [Hardware Requirements](#hardware-requirements)
 - [Networking Requirements](#networking-requirements)
 
 # Operating Systems and Docker Requirements
 
-For the nodes in user clusters, the requirements for the operating system and Docker version are the same as the [OS and Docker requirements for the Rancher server cluster.]({{<baseurl>}}/rancher/v2.x/en/installation/requirements/#operating-systems-and-docker-requirements)
+Rancher should work with any modern Linux distribution and any modern Docker version. Linux is required for the etcd and controlplane nodes of all downstream clusters. Worker nodes may run Linux or [Windows Server.](#requirements-for-windows-nodes) The capability to use Windows worker nodes in downstream clusters was added in Rancher v2.3.0.
+
+Rancher works has been tested with downstream clusters running Ubuntu, CentOS, Oracle Linux, RancherOS, and RedHat Enterprise Linux. For details on which OS and Docker versions were tested with each Rancher version, refer to the [support maintenance terms.](https://rancher.com/support-maintenance-terms/all-supported-versions/rancher-v2.3.0/)
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/cluster-provisioning/node-requirements/_index.md]: `Rancher has been tested and is supported with`

Should we actually refer to a specific version in the link?

File: content/rancher/v2.x/en/cluster-provisioning/node-requirements/_index.md [hunk 2]
```diff
@@ -7,17 +7,37 @@ This page describes the requirements for the nodes where your apps and services
...
+
+For information on how to install Docker, refer to the offical [Docker documentation.](https://docs.docker.com/)
+
+> **Note:** Some distributions of Linux derived from RHEL, including Oracle Linux, may have default firewall rules that block communication with Helm. This [how-to guide]({{<baseurl>}}/rancher/v2.x/en/installation/options/firewall) shows how to check the default firewall rules and how to open the ports with `firewalld` if necessary.
+
+### Requirements for Windows Nodes
+
+_Windows worker nodes can be used as of Rancher v2.3.0_
+
+Nodes with Windows Server core version 1809 should use Docker EE-basic 18.09 (1809) or Docker EE-basic 19.03 (1903).
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/cluster-provisioning/node-requirements/_index.md]: These requirements are also in the support matrix, so should we list it out?

File: content/rancher/v2.x/en/installation/requirements/_index.md [hunk 3]
```diff
@@ -24,24 +24,17 @@ For a list of best practices that we recommend for running the Rancher server in
 
 # Operating Systems and Docker Requirements
 
-Rancher is tested on the following operating systems and their subsequent non-major releases with a supported version of [Docker](https://www.docker.com/).
+Rancher should work with any modern Linux distribution and any modern Docker version.
 
-For details on which OS and Docker versions were tested with each Rancher version, refer to the [support maintenance terms.](https://rancher.com/support-maintenance-terms/all-supported-versions/rancher-v2.3.0/)
+Rancher has been tested with Ubuntu, CentOS, Oracle Linux, RancherOS, and RedHat Enterprise Linux.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/installation/requirements/_index.md]: `Rancher has been tested and is supported with`

File: content/rancher/v2.x/en/installation/requirements/_index.md [hunk 4]
```diff
@@ -24,24 +24,17 @@ For a list of best practices that we recommend for running the Rancher server in
...
 
-Operating System | Tested Docker Versions
----------------------|--------------------------
-Ubuntu 16.04 (64-bit x86) | Docker 17.03.x, 18.06.x, 18.09.x, 19.03.x
-Ubuntu 18.04 (64-bit x86) | Docker 18.06.x, 18.09.x, 19.03.x
-Red Hat Enterprise Linux (RHEL)/CentOS 7.7 (64-bit x86) | RHEL Docker 1.13
-Oracle Linux 7 update 6* (64-bit x86) | Docker 17.03.x, 18.06.x, 18.09.x, 19.03.x
-RancherOS 1.5.4 (64-bit x86) | Docker 17.03.x, 18.06.x, 18.09.x, 19.03.x
-Windows Server 2019 (64-bit x86) | Requires Docker Engine - Enterprise Edition (EE).** 
+For details on which OS and Docker versions were tested with each Rancher version, refer to the [support maintenance terms.](https://rancher.com/support-maintenance-terms/all-supported-versions/rancher-v2.3.0/)
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/installation/requirements/_index.md]: Same comment about is this the right link.

File: content/rancher/v2.x/en/cluster-provisioning/node-requirements/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/cluster-provisioning/node-requirements/_index.md]: updated wording and changed link to https://rancher.com/support-maintenance-terms/

File: content/rancher/v2.x/en/cluster-provisioning/node-requirements/_index.md (hunk 2 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/cluster-provisioning/node-requirements/_index.md]: No, it should probably just be maintained in the support terms. I'm removing the specific versions

File: content/rancher/v2.x/en/installation/requirements/_index.md (hunk 3 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/installation/requirements/_index.md]: updated wording

File: content/rancher/v2.x/en/installation/requirements/_index.md (hunk 4 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/installation/requirements/_index.md]: updated link to https://rancher.com/support-maintenance-terms/

**Pull request merged**